qa/tasks/ceph_manager.py

```diff
@@ -1362,15 +1362,18 @@ def raw_cluster_cmd(self, *args, **kwargs):
         """
         Start ceph on a raw cluster.  Return count
         """
-        stdout = kwargs.pop('stdout', StringIO())
-        p = self.run_cluster_cmd(args=args, stdout=stdout, **kwargs)
-        return p.stdout.getvalue()
+        if kwargs.get('args') is None and args:
+            kwargs['args'] = args
+        kwargs['stdout'] = kwargs.pop('stdout', StringIO())
+        return self.run_cluster_cmd(**kwargs).stdout.getvalue()
```

Comment:
@rishabh-d-dave i have following failure
```
2021-03-05T12:43:40.856 ERROR:teuthology.run_tasks:Saw exception from tasks.
Traceback (most recent call last):
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_3082387bba74fcd24c9700593d10418152d53c97/teuthology/run_tasks.py", line 91, in run_tasks
    manager = run_one_task(taskname, ctx=ctx, config=config)
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_3082387bba74fcd24c9700593d10418152d53c97/teuthology/run_tasks.py", line 70, in run_one_task
    return task(**kwargs)
  File "/home/teuthworker/src/git.ceph.com_ceph-c_830a81f35451008b24f2cfe8a8973667c1522bbf/qa/tasks/fs.py", line 59, in clients_evicted
    assert mount.is_blocklisted()
  File "/home/teuthworker/src/git.ceph.com_ceph-c_830a81f35451008b24f2cfe8a8973667c1522bbf/qa/tasks/cephfs/mount.py", line 582, in is_blocklisted
    blocklist = json.loads(self.fs.mon_manager.raw_cluster_cmd("osd", "blocklist", "ls", "--format=json"))
AttributeError: 'NoneType' object has no attribute 'mon_manager'
2021-03-05T12:43:40.940 ERROR:teuthology.run_tasks: Sentry event: https://sentry.ceph.com/organizations/ceph/?query=7e747acdbce647d78c884e18a312be26
Traceback (most recent call last):
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_3082387bba74fcd24c9700593d10418152d53c97/teuthology/run_tasks.py", line 91, in run_tasks
    manager = run_one_task(taskname, ctx=ctx, config=config)
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_3082387bba74fcd24c9700593d10418152d53c97/teuthology/run_tasks.py", line 70, in run_one_task
    return task(**kwargs)
  File "/home/teuthworker/src/git.ceph.com_ceph-c_830a81f35451008b24f2cfe8a8973667c1522bbf/qa/tasks/fs.py", line 59, in clients_evicted
    assert mount.is_blocklisted()
  File "/home/teuthworker/src/git.ceph.com_ceph-c_830a81f35451008b24f2cfe8a8973667c1522bbf/qa/tasks/cephfs/mount.py", line 582, in is_blocklisted
    blocklist = json.loads(self.fs.mon_manager.raw_cluster_cmd("osd", "blocklist", "ls", "--format=json"))
AttributeError: 'NoneType' object has no attribute 'mon_manager'
```

see https://pulpito.ceph.com/kchai-2021-03-05_11:19:56-upgrade-wip-kefu-testing-2021-03-05-1428-distro-basic-gibba/5939024/

could you help take a look? my batch includes both #39775 and this PR. but i suspect the failure was caused by this one.

Reply:
It's fixed by this: https://github.com/ceph/ceph/pull/39814